gui-tk/minsky.tcl

```diff
@@ -704,6 +704,11 @@ proc logVarsOK {} {
 .menubar.edit add command -label "Cut" -command cut -accelerator $meta_menu-X
 .menubar.edit add command -label "Copy" -command "minsky.copy" -accelerator $meta_menu-C
 .menubar.edit add command -label "Paste" -command "minsky.paste" -accelerator $meta_menu-V
+if {![catch {clipboard get -displayof .menubar.edit -type UTF8_STRING}]} {
```

Comment:
This has the same problem as I earlier mentioned. You need to do this at post time. There's no examples of doing this in the codebase, but if you consult the menu man page, you'll see that you should be able to attach a handler to run at post time via the -postcommand menu creation option.

Reply:
> you should be able to attach a handler to run at post time via the -postcommand menu creation option

This is where I'm stuck at present, the man page unfortunately does not specify what type of handler to pass to -postcommand.

Reply:
Any TCL command

Reply:
Understood, thanks.